gflownet/gflownet.py

```diff
@@ -754,10 +775,10 @@ def train(self):
             states_term, trajs_term = self.unpack_terminal_states(batch)
             proxy_vals = self.env.reward2proxy(rewards).tolist()
             rewards = rewards.tolist()
-            self.buffer.add(states_term, trajs_term, rewards, proxy_vals, it)
-            self.buffer.add(
-                states_term, trajs_term, rewards, proxy_vals, it, buffer="replay"
-            )
+            # self.buffer.add(states_term, trajs_term, rewards, proxy_vals, it)
+            # self.buffer.add(
+            #     states_term, trajs_term, rewards, proxy_vals, it, buffer="replay"
+            # )
```

Comment:
This should be uncommented, right?

Reply:
yeah that's an artifact from the memory leak debugging